gwr/gwr.py

```diff
@@ -846,6 +846,14 @@ def llnull(self):
     def null_deviance(self):
         raise NotImplementedError('Not implemented for GWR')
 
+    @cache_readonly
+    def R2(self):
+        if isinstance(self.family, Gaussian):
+            TSS = np.sum((self.y.reshape((-1,1)) - np.mean(self.y.reshape((-1,1))))**2)
+            RSS = np.sum((self.y.reshape((-1,1)) -
+                self.predy.reshape((-1,1)))**2)
+            return 1 - (RSS / TSS)
```

Comment:
Perhaps adding an else statement for non-Gaussian models?
Like for localR2
```
else:
    raise NotImplementedError('Only applicable to Gaussian')
```

Reply:
right now, this would only return `None` in that case. good catch.

Reply:
Yup, good call. Will now raise an error.